packages/website/docusaurus.config.js

```diff
@@ -105,21 +104,37 @@ const config = {
       prism: {
         theme: lightCodeTheme,
         darkTheme: darkCodeTheme,
+        additionalLanguages: ['ignore'],
       },
       tableOfContents: {
         maxHeadingLevel: 4,
         minHeadingLevel: 2,
       },
     }),
+  // Misleading API name, but these are just <link> tags
```

Comment:
Out of curiosity, is there a tracking Docusaurus issue about improving the API name?

Reply:
No, there isn't🤷‍♂️ I would try to make one yeah